### ISSUE TITLE
[Meta] 7.12 release

### ISSUE THREAD

**Issue by Leont** [Perl-Toolchain-Gang/ExtUtils-MakeMaker#252]
What is left to do before we can release a 7.12?

**Comment by bingos** [comment]
I unfortunately dropped the ball on this.

There are two issues as I recall.
And rt.cpan.org is not responding at the moment to link the salient tickets (502 Proxy Error)

Hopefully it'll be back later on for me to add the URLs.

**Comment by mohawk2** [comment]
#253 addresses #247.

**Comment by Leont** [comment]
> There is this one: https://rt.cpan.org/Ticket/Display.html?id=109962

I wouldn't be surprised if that started depending on the new -o behavior in the short time that such MakeMaker's were out (see [b6eb3c32](https://github.com/dk/Prima/commit/b6eb3c32aacd6bdad02e1a93bd6b26ba0fb47009)). Possibles the fixes could be to add XSMULTI to the Makefile.PL, or to revert that commit.

**Comment by mohawk2** [comment]
The Prima failure was due to its Makefile.PL relying on the generated Makefile having no space before the : in the rule in the c_o override. The pull request above removes that reliance.

**Comment by Leont** [comment]
I think both of these issues are resolved now, is there anything left that's keeping us from a 7.12 release?

**Comment by karenetheridge** [comment]
https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/issues/215 is marked as "blocking stable". I took a brief look at it a while ago and I think it's straightforward to do, for someone who knows the right bits of code to be modified.

**Comment by Leont** [comment]
I'm not sure why that would be a blocking issue. We've never done automatic minimum version detection before, why would it be required now? Not that it wouldn't be nice to have, but doing it only for this one feature seems a bit silly to me.

**Comment by karenetheridge** [comment]
@leont version range support is new. before, they were never handled properly (they were interpreted as '0'), but now that new EUMM versions support them, they might start being used in distributions. I think this is different from other features (e.g. TEST_REQUIRES) in that those other options would simply be ignored by earlier EUMMs (with a visible warning), but a version range will be silently accepted but turned into something that the author didn't intend.  So my contention was that some sort of backcompat handling should have been done before the original feature (version range support) was ever shipped as stable.

**Comment by mohawk2** [comment]
Following the discussion of #260 it seems as though that's not now a blocker. That implies to me that 7.11_05 is an RC for 7.12? Or is there more to do?

**Comment by haarg** [comment]
#260 is definitely not a blocker.

**Comment by bingos** [comment]
Shipped.
